Changelog

```diff
@@ -1,4 +1,81 @@
 Unreleased - sledgehammer999 <sledgehammer999@qbittorrent.org> - v4.5.0
+    - FEATURE: Add `Auto resize columns` functionality (Chocobo1)
+    - FEATURE: Allow to use Category paths in `Manual` mode (glassez)
+    - FEATURE: Allow to disable Automatic mode when default "temp" path changed (glassez)
+    - FEATURE: Add tuning options related to performance warnings (Chocobo1)
+    - FEATURE: Add right click menu for status filters (An0n)
+    - FEATURE: Allow setting the number of maximum active checking torrents (An0n)
+    - FEATURE: Add option to toggle filters sidebar (AbeniMatteo)
+    - FEATURE: Allow to set `working set limit` on non-Windows OS (Chocobo1)
+    - FEATURE: Add `Export .torrent` action (Chocobo1)
+    - FEATURE: Add keyboard navigation keys (itlezy)
+    - FEATURE: Allow to use POSIX-compliant disk IO type (Coda)
+    - FEATURE: Add `Filter files` field in new torrent dialog (thalieht)
+    - FEATURE: Implement new icon/color theme (now-im)
+    - FEATURE: Add file name filter/blacklist (mxtsdev, thalieht)
+    - FEATURE: Add support for custom SMTP ports (Emil M George)
+    - FEATURE: Split the OS cache settings into Disk IO read/write modes (summer)
+    - FEATURE: When duplicate torrent is added set metadata to existing one (glassez)
+    - FEATURE: Greatly improve startup time with many torrents (glassez)
```

Comment:
I've also done substantial work on this. https://github.com/qbittorrent/qBittorrent/pull/16587 https://github.com/qbittorrent/qBittorrent/pull/16357 https://github.com/qbittorrent/qBittorrent/pull/16329 https://github.com/qbittorrent/qBittorrent/pull/16298

Reply:
> I've also done substantial work on this. #16587 #16357 #16329 #16298

The startup process has been heavily redesigned since then, so it's unlikely that all of them have a tangible impact on the startup time. #16357 and #16329 apparently still affect. #16587 is unlikely, but it still has an impact when using GUI. As for #16298, there is hardly anything significant left of it in the current code.
So they could be mentioned separately.

Reply:
maybe add @jagannatharjun too in the attribution list for this line?

Reply:
These are all different kind of changes, so I would prefer that they were mentioned separately. I don't want to assign myself all the contribution to improving the startup time, so my contribution can be named differently, e.g. `Revamp startup process`.

Reply:
`Revamp startup process` or `Do RSS serializing on worker thread` or `Optimize handleFileCompletedAlert` or `Optimize torrent filters in GUI` mean nothing to the end user.
The changelog should reflect what the user will see or experience. Unless I am mistaken your PR and @jagannatharjun 's PRs affect qbittorrent's startup time. That's what the user will experience. He doesn't care how we accomplished it internally. If he cares, the git log is public and shows exactly who did what with greater detail.

That being said, I also intend to mention in the `news` section that `4.5.0` has a lot of code refactorings by various contributors that result in general program optimization and better code architecture.

Reply:
>The changelog should reflect what the user will see or experience.

👍 
>your PR and @jagannatharjun 's PRs affect qbittorrent's startup time. That's what the user will experience.

OK, it's up to you. I won't argue.

Reply:
> That being said, I also intend to mention in the `news` section that `4.5.0` has a lot of code refactorings by various contributors that result in general program optimization and better code architecture.

It would be nice to mention our @qbittorrent/bug-handlers. And a special "thank you" from me to @thalieht - without his expeditious testing of my PRs, my work would be much less productive. 
I also want to thank @oorzkws for testing the "improve startup" stuff in circumstances of a really heavy load (about several hundred thousand torrents).